tooling/api/src/fs.ts

```diff
@@ -542,6 +543,30 @@ async function renameFile(
   })
 }
 
+/**
+ * Check if a path exists.
```

Comment:
or whatever version this PR lands in.

```suggestion
 * Check if a path exists.
 * @since 1.1.0
```